pl.lproj/Localizable.strings

```diff
@@ -0,0 +1,160 @@
+"EDIT_STATION_WINDOW_IMAGE_ACCESSIBILITY_LABEL" = "Wybierz Okładkę";
+
+"ADD_COLLECTION_WINDOW_TITLE" = "Dodaj Nową Kolekcję";
+"EDIT_COLLECTION_WINDOW_TITLE" = "Edytuj Kolekcję";
+
+"EDIT_COLLECTION_WINDOW_NAME_PLACEHOLDER" = "Moja Kolekcja";
+
+"SIDEBAR_CONTEXT_EDIT_COLLECTION" = "Edytuj";
+"SIDEBAR_CONTEXT_DELETE_COLLECTION" = "Usuń Kolekcję";
+"SIDEBAR_CONTEXT_SHARE_COLLECTION" = "Udostępnij";
+
+"STATION_CONTEXT_EDIT_COLLECTION" = "Edytuj";
+"STATION_CONTEXT_DELETE_COLLECTION" = "Usuń Stację";
+"STATION_CONTEXT_MOVE" = "Umieść w Kolekcji";
+
+"CONTEXT_ALERT_DELETE_COLLECTION_TITLE" = "Usuń \"%@\"";
+"CONTEXT_ALERT_DELETE_COLLECTION_BODY" = "Czy na pewno chcesz usunąć tę kolekcję i wszystkie powiązane stacje?";
+
+"CONTEXT_ALERT_DELETE_STATION_TITLE" = "Usuń \"%@\"";
+"CONTEXT_ALERT_DELETE_STATION_BODY" = "Czy na pewno chcesz usunąć tę stację?";
+
+"COLLECTION_VIEW_EMPTY_SECTION_TITLE" = "Brak Stacji";
+
+"WELCOME_COPY" = "Witamy w Broadcasts! Dodaliśmy już parę naszych Irlandzkich stacji radiowych, aby twoja biblioteka nie była pusta.\n\nJeżeli wolisz, możesz dodać ręcznie własne stacje lub wybrać którąś z tysiąca stacji w naszym katalogu.";
+
+"WELCOME_UPSELL" = "Witamy w Broadcasts! W tej darmowej wersji dodaliśmy już parę naszych Irlandzkich stacji radiowych, aby biblioteka nie była pusta.\n\nJeżeli przejdziesz do płatnej wersji, możliwe będzie dodanie ręcznie własnych stacji lub wybranie którejś z tysiąca stacji w naszym katalogu.";
+
+"WELCOME_ADD_PRESETS" = "Dodaj Stacje Początkowe";
+"WELCOME_NO_THANKS" = "Nie, dziękuję";
+
+"GENERATED_COLLECTION_TITLE" = "Moja Kolekcja";
+
+"UPSELL_COPY" = "Dzięki za korzystanie z Broadcasts!\n\nJeżeli chcesz, możesz odblokować zaawansowane funkcje zakupem w aplikacji.\n\nJeżeli wolisz, możesz po prostu edytować lub korzystać dalej ze stacji początkowych!";
+"UPSELL_BUY" = "Kup (%@)";
+"UPSELL_NOT_AVAILABLE" = "Zakup Nie Dostępny";
+"UPSELL_RESTORE" = "Odnów Zakup";
+"UPSELL_NO_THANKS" = "Nie, Dziękuję";
+
+"TOOLBAR_NEW_CATEGORY_TOOLTIP" = "Nowa Kolekcja";
+"TOOLBAR_ADD_STATION_TOOLTIP" = "Dodaj Stację";
+"TOOLBAR_SHOW_SOURCES_TOOLTIP" = "Pokaż Kolekcje";
+"TOOLBAR_TOGGLE_SIDEBAR_TOOLTIP" = "Przełącz Pasek Boczny";
+"TOOLBAR_TOGGLE_VIEW_TOOLTIP" = "Przełącz Widok";
+
+"IMAGE_WELL_REMOVE" = "Usuń";
+"IMAGE_WELL_PASTE" = "Wklej";
+"IMAGE_WELL_COPY" = "Kopiuj";
+
+"CARPLAY_EMPTY_LIBRARY" = "Brak Kolekcji";
+"CARPLAY_ALL_STATIONS" = "Wszystkie Stacje";
+
+"MAC_ADD_STATION_BUTTON" = "Dodaje Stację";
+"MAC_EDIT_STATION_BUTTON" = "Edytuj Stację";
+
+"MAC_ADD_COLLECTION_BUTTON" = "Dodaj Kolekcję";
+"MAC_EDIT_COLLECTION_BUTTON" = "Edytuj Kolekcję";
+
+"STATION_DIRECTORY_WINDOW_TITLE" = "Przeglądaj Stacje"
+"STATION_DIRECTORY_COUNTRIES_TITLE" = "Kraje";
+"STATION_DIRECTORY_STATIONS" = "Stacja %@";
+"STATION_DIRECTORY_STATIONS_PLURAL" = "Stacje %@";
+
+"STATION_DIRECTORY_NO_CONNECTION" = "Brak Połączenia";
+"STATION_DIRECTORY_CONNECTING" = "Ładuję…";
```

Comment:
If STATION_DIRECTORY_CONNECTING means that something's seeking a connection to something else, then it should be "Łączę..." instead of "Ładuję..." (which means "loading").

Reply:
Loading in this case I think might be the right word; it's loading the station directory, but it can also report no connection to the internet/server. I'm happy to go with loading

Reply:
Sounds like the right choice.

Reply:
morid1n, thanks for the corrections!

Reply:
@chdamian YW and thanks for doing all the initial work, since it made it easier for me. 😅